annlite/index.py

```diff
@@ -621,6 +618,32 @@ def snapshot_path(self):
             return paths[0]
         return None
 
+    @property
+    def remote_store(self):
+        try:
+            import hubble
+
+            client = hubble.Client(max_retries=None, jsonify=True)
+            client.get_user_info()
+            return client
+        except hubble.excepts.AuthenticationRequiredError:
+            print('Please login first.')
```

Comment:
Please use `logger.error(...)` to report the message. And also please raise error exception so that the executor can catch this exception and throw it into the response.